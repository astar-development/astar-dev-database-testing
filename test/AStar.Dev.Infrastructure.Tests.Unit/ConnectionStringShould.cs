using AStar.Dev.Infrastructure.Data;
using AStar.Dev.Utilities;
using Shouldly;

namespace AStar.Dev.Infrastructure.Tests.Unit;

public class ConnectionStringShould
{
    [Fact]
    public void ContainTheExpectedProperties()
        => new ConnectionString { Value = "Clearly, this is not a valid connection string" }
           .ToJson()
           .ShouldMatchApproved();
}
