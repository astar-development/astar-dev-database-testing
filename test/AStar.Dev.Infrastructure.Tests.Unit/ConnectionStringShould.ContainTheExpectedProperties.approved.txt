{
  "value": "Clearly, this is not a valid connection string"
}